json-schema/schema.json

```diff
@@ -160,46 +162,24 @@
     "require_any_field": {
       "$comment": "Please list all fields here so that we can force the existence of one of them in other parts of the schemas.",
       "anyOf": [
-        {"required": ["template:new_field"]},
-        {"required": ["template:xyz"]},
-        {"required": ["template:another_one"]}
+        {"required": ["marble:host_node"]},
+        {"required": ["marble:is_local"]}
       ]
     },
     "fields": {
-      "$comment": "Add your new fields here. Don't require them here, do that above in the corresponding schema.",
       "type": "object",
       "properties": {
-        "template:new_field": {
+        "marble:host_node": {
+          "title": "Marble node hosting this data",
```

Comment:
Should we clarify that this should be the host's ID according to the network registry (not its display name or some other identifier).

Reply:
Yeah, we should that to README.

Reply:
Inclusion of the node-name validation pattern sounds like a good idea.